cluster-scope/base/operators.coreos.com/subscriptions/openshift-cnv/subscription.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: hco-operatorhub
+  namespace: openshift-cnv
```

Comment:
I think this should be named `kubevirt-hyperconverged` to match the name of the operator.

Reply:
changed.